File: transform.js
const fs=require("fs");
const fp="d:/HobbyProjects/collm/apps/web/app/nodes/[id]/page.tsx";
let c=fs.readFileSync(fp,"utf8");
c=c.replace(/\r\n/g,"\n");
console.log("len:",c.length);
fs.writeFileSync("d:/HobbyProjects/collm/transform.js",lines.join("\n"),"utf8");

File: transform.py
fp=r"d:\\HobbyProjects\\collm\\apps\\web\\app\\nodes\\[id]\\page.tsx"
with open(fp,"r",encoding="utf-8") as f:c=f.read()
ok=[];fail=[]

## STEP 1: Add detectContested
old1="""// ---------------------------------------------------------------------------\n// Expandable contribution list item (4 lines at a time)\n// ---------------------------------------------------------------------------"""
